src/components/dashjs-api-control-element/dashjs-api-control-element.css

```diff
@@ -1,3 +1,14 @@
 :host {
   display: block;
 }
+
+.middle {
+  display: table-cell;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
```

Comment:
You can reduce the margin with 
https://github.com/AWTPJ2021/dash-reference-ui/blob/06a1280a0914a031457c52e719bead075c12a9cc/src/components/dashjs-settings-control-element/dashjs-settings-control-element.css#L1-L5

Reply:
its to vertical align the text with the call button

Reply:
? thats the way how to remove padding from both